File: satva/src/app/customer.ts
import { Address } from "./address";

export class Customer {    
    private firstName: string = "";
    public get firstname(): string {
        return this.firstName;
    }
    public set firstname(value: string) {
        this.firstName = value;
    }
	private lastName: string = "";
    public get lastname(): string {
        return this.lastName;
    }
    public set lastname(value: string) {
        this.lastName = value;
    }
	private emailId: string = "";
    public get emailid(): string {
        return this.emailId;
    }
    public set emailid(value: string) {
        this.emailId = value;
    }
	private passWord: string = "";   
    public get password(): string {
        return this.passWord;
    }
    public set password(value: string) {
        this.passWord = value;
    }
    private _address: Address = new Address();
    public get address(): Address {
        return this._address;
    }
    public set address(value: Address) {
        this._address = value;
    }
    
    
   
}

File: demo/src/main/java/com/example/demo/beans/Customer.java
package com.example.demo.beans;



public class Customer {
	
	private String firstName;
	private String lastName;
	private String emailId;
	private String passWord;
	private Address address;
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	public String getEmailId() {
		return emailId;
	}
	public void setEmailId(String emailId) {
		this.emailId = emailId;
	}
	public String getPassWord() {
		return passWord;
	}
	public void setPassWord(String passWord) {
		this.passWord = passWord;
	}
	public Address getAddress() {
		return address;
	}
	public void setAddress(Address address) {
		this.address = address;
	}
}
